deploy/assisted-installer-controller/assisted-installer-controller-pod.yaml.template

```diff
@@ -56,3 +61,11 @@ spec:
       - key: node-role.kubernetes.io/master
         effect: NoSchedule
         operator: Exists
+      {{if .CACertPath}}
+      volumes:
+      - name: service-ca-cert-config
+        hostPath:
+          path: {{.CACertPath}}
```

Comment:
If this is going to be `/etc/assisted-service/<something>` should the path in the container be different than this one or will that path work somehow?

Reply:
The env variable CA_CERT_PATH is set to /etc/assisted-service/service-ca-cert.crt. And this just hostPaths the volume /etc/assisted-service/service-ca-cert.crt:/etc/assisted-service/service-ca-cert.crt to the container.

Reply:
Yes, I understand that. My question is if `/etc/assisted-service/service-ca-cert.crt` is the correct path to use in the container or should it be placed somewhere else?

Reply:
/etc/pki could be the obvious one. But needs privileged access for the hostpath to be shared, which is really unnecessary. Outside of that, /etc/assisted-service sounds like a reasonable place. Could be any path really.

Reply:
:+1: didn't see this in the env list